<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QWidget" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>835</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Слепопечать v0.1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#main_menu {
	background-color: transparent;
	background: none;
    border: none;
	background-repeat: none;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3,3,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="main_logo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>650</width>
         <height>325</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#main_logo {
background-color: transparent;
border-image:url(:/images/images/big_logo.png);
background: none;
border: none;
background-repeat: none;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="startgame_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>325</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>30</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">#startgame_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 30pt;
}

#startgame_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 28pt;
}</string>
       </property>
       <property name="text">
        <string>Начать игру</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="record_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>325</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">#record_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 30pt;
}

#record_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 28pt;
}</string>
       </property>
       <property name="text">
        <string>Рекорды</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="quit_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>325</width>
         <height>75</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>325</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">#quit_button {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 30pt;
}

#quit_button:pressed {
    background: transparent;
    font: &quot;Comic Sans MS&quot;;
    font-size: 28pt;
}</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="settings_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#settings_button {
background-color: transparent;
border-image: url(:/icons/images/settings.png);
background: none;
border: none;
background-repeat: none;
}
#settings_button:pressed
{
   border-image: url(:/icons/images/settings_clicked.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc_files/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
